.buildkite/test-pipeline.yaml

```
@@ -396,6 +396,21 @@ steps:
     - pytest -v -s tensorizer_loader
     - pytest -v -s entrypoints/openai/test_tensorizer_entrypoint.py
 
+- label: Model Executor Test
+  mirror_hardwares: [amdexperimental, amdproduction]
+  soft_fail: true
+  source_file_dependencies:
+  - vllm/model_executor
+  - tests/model_executor/test_model_load_with_params
+  - tests/model_executor/test_enabled_custom_ops
+  - tests/model_executor/test_weight_utils
+  commands:
+    - apt-get update && apt-get install -y curl libsodium23
+    - export VLLM_WORKER_MULTIPROC_METHOD=spawn
+    - pytest -v -s model_executor/test_model_load_with_params.py
+    - pytest -v -s model_executor/test_enabled_custom_ops.py
+    - pytest -v -s model_executor/test_weight_utils.py
```

```suggestion
    - pytest -v -s model_executor
```

We can simply run all files

---

tests/model_executor/test_guided_processors.py is already there in another step.